python/lsst/cp/pipe/defects.py

```diff
@@ -190,7 +208,7 @@ def _nPixFromDefects(defects):
             nPix += defect.getBBox().getArea()
         return nPix
 
-    def findHotAndColdPixels(self, exp, nSigma):
+    def findHotAndColdPixels(self, exp):
```

Comment:
A small thing, but if this is not a public interface (and if we can change the interface like this I hope not!) then we can make it `_findHotAndColdPixels`.